### ISSUE TITLE
Screen Readers: Create Expenses: Recipient information is announced without the second line of text

### ISSUE THREAD

**Issue by olenamihno** [Expensify/App#77449]
## Action Performed:
Precondition: The User should be logged in

1. Using Android TalkBack or iOS VocieOver, open the Expensify app
2. Swipe to the Create '+' button and double-tap to activate
3. Swipe to the 'Create Expense' button and double-tap to activate
4. Type the expense amount, swipe to the 'Next' button, and double-tap to activate
5. Swipe to the 'Personal' button
6. Note how the control is announced.

## Expected Result:
The expected result is a single, clear, and complete announcement upon focus: "[Recipient Name], [Second Line Detail]."

## Actual Result:
When a screen reader user focuses on the recipient block, it announces only the first line of text and then stops, omitting the second line.

## Area issue was found in:
Create Expenses

## Failed WCAG checkpoints
WCAG 2.4.6 Headings and Labels
https://www.w3.org/WAI/WCAG22/Understanding/headings-and-labels

## User impact:
Descriptive headings are necessary for users who have disabilities that make reading slow and for people with limited short-term memory. These users benefit when section titles make it possible to predict what each section contains. Form input controls with labels that clearly describe the content that is expected to be entered helps users know how to successfully complete the form.

## Suggested resolution:
Consolidate both lines of text into the accessibilityLabel.
https://reactnative.dev/docs/button#accessibilitylabel

## Environment:
Samsung Galaxy S24 Ultra - Android 14, Partner - Native (No mobile browser)
Apple iPhone 11 - iOS 17.5.1, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos

![Image](https://github.com/user-attachments/assets/4becdb37-9bec-4fde-868f-a4153e16786c)

https://github.com/user-attachments/assets/33de63fe-758c-47a9-bfba-cd382a46ea9a

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** No
**Logs:** https://stackoverflow.com/c/expensify/questions/4856  NA
**Notes/Photos/Videos:** Any additional supporting documentation  Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7006722
**Issue reported by:** NA
**Slack conversation:** NA
